src/main/scala/tech/cryptonomic/conseil/tezos/michelson/dto/MichelsonInstruction.scala

```diff
@@ -32,8 +34,17 @@ case class MichelsonSingleInstruction(
 /* Class representing a sequence of Michelson instructions */
 case class MichelsonInstructionSequence(instructions: List[MichelsonInstruction] = List.empty)
     extends MichelsonInstruction {
-  lazy val normalized: MichelsonInstruction = if (instructions.isEmpty) MichelsonEmptyInstruction else this
+  override lazy val normalized: MichelsonInstruction = if (instructions.isEmpty) MichelsonEmptyInstruction else this
 }
 
+/* Class representing an int constant */
+case class MichelsonIntConstant(int: Long) extends MichelsonInstruction
```

Comment:
"Literal" instead of "Constant" would be better here.

Reply:
Probably yes, but in the docs, they use word "Constant".